### DISCUSSION TITLE
How to delete a file from a zipfile?

### DISCUSSION THREAD

**Discussion by jinfeihan57** [nmoinvaz/minizip#524] [Q&A] [answered]
See the title.

**Comment by nmoinvaz** [comment, marked as answer]
To remove an entry from a zip file, you can take a look at `minizip_erase` in `minizip.c`. It involves recreating the zip file without the specific entry you are deleting.

https://github.com/nmoinvaz/minizip/blob/3ecff07e3cf76b848b9d6b219302c1c5311aa281/minizip.c#L423
